__tests__/components/AnonymousChat/child/ChatItemTargetText.test.tsx

```diff
@@ -15,12 +15,19 @@ import {
   timestamp
 } from './ChatItemMyTextV2.test';
 
+global.__reanimatedWorkletInit = jest.fn();
 jest.spyOn(React, 'useRef').mockReturnValue({current: {close: jest.fn()}});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The global `__reanimatedWorkletInit` is being mocked here. This could potentially affect other tests that rely on the original implementation of `__reanimatedWorkletInit`. Consider moving this mock to a more specific test case where it's needed, or ensure to restore the original implementation after the tests.
```diff
- global.__reanimatedWorkletInit = jest.fn();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->